### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by ghost** [dhasirar/instax#26]
not working

**Comment by ghost** [comment]
tried with own account+pasword

**Comment by unknownX00X** [comment]
Hello these steps will help you 
1. PKG install tor 
2. PKG install curl
3. PKG install openssl

**Comment by gopal-spec** [comment]
yeah this toool is not working i put the password from the list itself then also it is not working
